types/aws-lambda/index.d.ts

```diff
@@ -1153,7 +1162,7 @@ export type SQSHandler = Handler<SQSEvent, void>;
 
 export type CloudFormationCustomResourceHandler = Handler<CloudFormationCustomResourceEvent, void>;
 
-// TODO: CloudWatchEvents
+export type EventBridgeHandler<T = object> = Handler<EventBridgeEvent<T>, void>;
```

Comment:
Don't have defaults here (unless they are on the event type as well), and I have similar suspicion on the detail type here as above: `object` is not the same type as `{}`, and has little reason to be used in most situations. It may well be the case that the type is never going to be a primitive (which is what `object` implies), but there's little value gained by defaulting that here.